installer/inventory

```diff
@@ -28,10 +28,11 @@ dockerhub_base=ansible
 # pg_cpu_limit=1000
 # pg_mem_limit=2
 
+# Ingress Configuratio
+ingress_hostname=awx.example.org
```

Comment:
Other default variables in this file are commented out. We should probably do that here too.